relay-crates/common/src/diagnostic_check.rs

```diff
@@ -156,7 +156,7 @@ mod escalate_tests {
 
         escalate(DiagnosticSeverity::WARNING, &mut diagnostics);
```

Comment:
The relay-crates folder contains forks of crates that actually exist in https://github.com/facebook/relay; if there's a good way to disable clippy for these folders, that would be better than modifying them (as I don't expect to modify the files in this folder at all.)

But if that's not easy, we can merge these changes no problem

You also may want to make these changes upstream to Relay! I'm sure they'd appreciate it, or (maybe) have some clippy config that causes this code to pass.

Reply:
Got it, let's disable them.